core/lib/dal/migrations/20240523085604_add_tee_proof_generation_details_table.up.sql

```diff
@@ -1,22 +1,25 @@
+CREATE TYPE tee_proofs_job_status AS ENUM ('ReadyToBeProven', 'PickedByProver', 'Generated', 'Skipped');
```

Comment:
When thinking about migrations, consider that there are non-empty tables in the existing envs with values in them. Would it work there?

Reply:
Same response as here: https://github.com/matter-labs/zksync-era/pull/2474#discussion_r1705158912.